src/cms/templates/pages/page_form.html

```diff
@@ -68,22 +68,23 @@ <h2 class="heading font-normal">
             {% endif %}
             {% has_perm 'cms.publish_page' request.user page as can_publish_page %}
             {% if can_publish_page %}
-                <input type="submit" name="submit_public" class="cursor-pointer bg-blue-500 hover:bg-blue-600 text-white font-bold py-3 px-4 rounded mr-2" value="{% trans 'Publish' %}" />
+                <input type="submit" name="submit_public" class="cursor-pointer bg-blue-500 hover:bg-blue-600 text-white font-bold py-3 px-4 rounded mr-2"
+                    value="{% if page %}{% trans 'Update' %}{% else %}{% trans 'Publish' %}{% endif %}" />
```

Comment:
Ok, I've updated the submit button, taking into account whether the page form is public or draft, see 897a62b785ef2d8d5644dbda2dda7281b4bddf2d

Reply:
Ok, I've updated the submit button, taking into account whether the page form is public or draft, see https://github.com/Integreat/integreat-cms/pull/733/commits/897a62b785ef2d8d5644dbda2dda7281b4bddf2d

Reply:
Since we also have the status ["Pending Review"](https://github.com/Integreat/integreat-cms/blob/b0594e0b4ad816029835b8f8188e69d3eead9908/src/cms/constants/status.py#L10) (in this case the button should also be called "Publish" instead of "Update"), I think it would make sense to check `page_translation_form.instance.status != PUBLIC` instead of `page_translation_form.instance.status == DRAFT`...

Reply:
Alright, this one 46d7ce65c955f514798adced7febbdc9a98a6280 should cover pending review state too